eva-pipeline/src/main/resources/application-production.properties

```diff
@@ -3,3 +3,10 @@ job.repository.driverClassName=org.postgresql.Driver
 job.repository.url=jdbc:postgresql://10.7.248.20:49153/spring_batch_annotations
 job.repository.username=postgres
 job.repository.password=batch
```

Comment:
Job repository credentials should be removed and values filled using Maven profiles.